node/yarn.lock

```diff
@@ -0,0 +1,1181 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
delete this file, we use npm instead of yarn

Reply:
done